dapp-oeth/src/components/layout.js

```diff
@@ -0,0 +1,197 @@
+import React from 'react'
+import classnames from 'classnames'
+import Head from 'next/head'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { useStoreState } from 'pullstate'
+import { fbt } from 'fbt-runtime'
+import { useWeb3React } from '@web3-react/core'
+import { get } from 'lodash'
+import AccountStore from 'stores/AccountStore'
+import ContractStore from 'stores/ContractStore'
+import StakeStore from 'stores/StakeStore'
+import withRpcProvider from 'hoc/withRpcProvider'
+import AppFooter from './AppFooter'
+import { adjustLinkHref } from 'utils/utils'
+import { assetRootPath } from 'utils/image'
+import { burnTimer } from 'utils/constants'
+
+const UNISWAP_URL =
```

Comment:
nit: Update to point to OETH/ETH pool on Uniswap V3? However, this notice is hidden at the moment I guess